src/ceph_osd.cc

```diff
@@ -73,6 +73,10 @@ static void usage()
        << "  --osd-journal PATH\n"
        << "                    journal file or block device\n"
        << "  --mkfs            create a [new] data directory\n"
+       << "  --mkkey           generate a new secret key\n"
+       << "  --monmap          specify the path to the monitor map\n"
+       << "  --osd-uuid        specify the OSD's fsid\n"
+       << "  --keyring         specify a path to the osd keyring\n"
```

Comment:
These are specifically relevant to --mkfs, so it seems ok to manually include them here?